Dockerfile

```diff
@@ -325,6 +374,21 @@ FROM binaries AS buildkit-windows
 # this is not in binaries-windows because it is not intended for release yet, just CI
 COPY --link --from=buildkitd /usr/bin/buildkitd /buildkitd.exe
 
+# can be removed when binaries for windows are released
+FROM binaries AS buildkit-binaries-linux
+FROM binaries AS buildkit-binaries-darwin
+FROM binaries AS buildkit-binaries-freebsd
+FROM scratch AS buildkit-binaries-windows
```

Comment:
There seems to be some duplication in here as there already is `binaries-windows` stage.